src/main/java/com/zerocracy/pmo/banks/Zold.java

```
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2016-2018 Zerocracy
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to read
+ * the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.zerocracy.pmo.banks;
+
+import com.zerocracy.cash.Cash;
+import java.io.IOException;
+
+/**
+ * Zold payment.
+ * @author Kirill (g4s8.public@gmail.com)
+ * @version $Id$
+ * @since 0.22
+ * @todo #1074:30min Zold payments are not implemented, we have to modify
+ *  wallet by including new transaction for user and push it to some node
+ *  via restful API: PUT /wallet/ID.
+ */
+public final class Zold implements Bank {
+    @Override
+    public Cash fee(final Cash amount) throws IOException {
+        return Cash.ZERO;
+    }
+
+    @Override
+    public String pay(final String target, final Cash amount,
+        final String details) throws IOException {
+        throw new IOException("Zold payments are not implemented yet");
```

@g4s8 Let's throw `UnsupportedOperationException` instead, to easily distinguish whether we're calling this accidentally when we shouldn't yet.